android/301Project/app/src/main/java/com/example/a301project/AddEditRecipeFragment.java

```diff
@@ -30,6 +30,7 @@
 import androidx.fragment.app.DialogFragment;
 import androidx.fragment.app.Fragment;
 
+<<<<<<< HEAD
```

Comment:
Should prolly take these <<<Head things out ?

Reply:
yea Im actually not sure what those are, I didn't intentionally add them